src/main/kotlin/no/nav/syfo/persistering/db/PersisterDialogmeldingQueries.kt

```diff
@@ -326,6 +341,32 @@ fun DatabaseInterface.hasSavedDialogmeldingDokument(dialogmeldingId: String, sha
         }
     }
 
+fun DatabaseInterface.getUnpublishedArenaMeldinger(): List<Pair<String, String>> =
+    connection.use { connection ->
+        connection.prepareStatement(
+            """
+            SELECT id, fellesformat
+            FROM dialogmeldingopplysninger d
+            WHERE arena IS NULL
+                AND apprec IS NOT NULL
+                AND apprec < (NOW() - INTERVAL '10 minutes')
+                AND dialogmelding_published IS NOT NULL
+                AND EXISTS (
+                    SELECT 1
+                    FROM behandlingsutfall b
+                    WHERE b.id = d.id AND behandlingsutfall ->> 'status' = 'OK'
+                );
+            """
```

Comment:
`behandlingsutfall` er en tabell med kolonner `id` og `behandlingsutfall`, der sistnevnte er en `jsonb`-struktur. 
Her er litt om syntaks for å hente ut info fra en slik en: https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-json/